From 9de64bc0c1857e7b504bab874e1b7ef775477e2f Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Sun, 10 Jul 2022 22:56:53 -0400
Subject: lib/build_OID_registry: fix reproducibility issues

The script build_OID_registry captures the full path of itself
in the generated data. This causes reproduciblity issues as the
path is captured and packaged.

We use the basename of the script instead, and that allows us
to be reprodicible, with slightly less information captured in
the output data (but the generating script can still easily
be found).

Upstream-Status: Inappropriate [Yocto-specific]
Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 lib/build_OID_registry | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

(limited to 'lib/build_OID_registry')

diff --git a/lib/build_OID_registry b/lib/build_OID_registry
index d7fc32ea8ac2..f6de0a7f7457 100755
--- a/lib/build_OID_registry
+++ b/lib/build_OID_registry
@@ -8,6 +8,7 @@
 #
 
 use strict;
+use File::Basename;
 
 my @names = ();
 my @oids = ();
@@ -35,7 +36,7 @@ close IN_FILE || die;
 #
 open C_FILE, ">$ARGV[1]" or die;
 print C_FILE "/*\n";
-print C_FILE " * Automatically generated by ", $0, ".  Do not edit\n";
+print C_FILE " * Automatically generated by ", basename $0, ".  Do not edit\n";
 print C_FILE " */\n";
 
 #
-- 
cgit 1.2.3-korg

